app/products/calls/actions/calls.ts

```diff
@@ -263,3 +275,97 @@ export const setSpeakerphoneOn = (speakerphoneOn: boolean) => {
     InCallManager.setSpeakerphoneOn(speakerphoneOn);
     setSpeakerPhone(speakerphoneOn);
 };
+
+export const endCallAlert = async (serverUrl: string, intl: IntlShape, channelId: string) => {
+    const database = DatabaseManager.serverDatabases[serverUrl]?.database;
+    if (!database) {
+        return;
+    }
+
+    const currentUser = await getCurrentUser(database);
+    if (!currentUser) {
+        return;
+    }
+
+    const channel = await getChannelById(database, channelId);
+    if (!channel) {
+        return;
+    }
+
+    const call = getCallsState(serverUrl).calls[channelId];
+    if (!call) {
+        return;
+    }
+
+    const numParticipants = Object.keys(call.participants).length || 0;
+    const isAdmin = isSystemAdmin(currentUser.roles);
+    const {formatMessage} = intl;
+
+    if (!isAdmin && currentUser.id !== call.creatorId) {
```

Comment:
We could, but then the error is not as polite and not internationalizable. 1/5 I'd keep it like this, but open to changing.

Reply:
Are we using that variable anywhere else? What I am suggesting is simply to avoid the initialization and call `isSystemAdmin` directly since it's clear what it does. Very minor thing though :)

Reply:
@streamer45 Ah, gotcha. :) Fixed in: a413cfbc6836d7a107c9136889075a05c5dd57b0